src/main.cc

```diff
@@ -77,6 +77,21 @@ int wmain(int argc, const wchar_t* argv[]) {
 
       if (!updater.SetIcon(argv[++i]))
         return print_error("Unable to set icon");
+    } else if (wcscmp(argv[i], L"--set-requestedExecutionLevel") == 0 || 
```

Comment:
Oh, I assumed you were going to use `--set-requested-execution-level` - that seems like the more consistent name. It's not `--set-fileVersion`, for example.